consumers/notifiers/api/transport.go

```diff
@@ -21,7 +21,11 @@ import (
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
-const contentType = "application/json"
+const (
+	contentType = "application/json"
+
```

Comment:
No need for an empty line. Please extract `offset` and `limit` keys to constants, it's like that in the rest of the codebase.